### DISCUSSION TITLE
Firejail and Steam games on Ubuntu.

### DISCUSSION THREAD

**Discussion by opqriu** [netblue30/firejail#4458] [Q&A] [answered]
Firejail and Steam games on Ubuntu.

Hello?
Games that use Proton will not run on firejail.
However, if the --noprofile option is given, it runs without any problem. (Steam Proton Issue)
I'd love to hear from you if anyone has a pioneer in solving this problem.

Thank You.
Good day.

**Comment by rusty-snake** [comment, marked as answer]
If you use firejail < 0.9.65: https://github.com/netblue30/firejail/commit/3a7deadb9a0cce605f1d1a79a05653ec1676a8bf

----

My first guess would be seccomp
https://github.com/netblue30/firejail/blob/5f9873d454fa7b028a3433bf94928c78bcc02633/etc/templates/syscalls.txt#L89-L112

- firejail < 0.9.64: There is no `--seccomp-error-action=log`
- firejail < 0.9.62: There is no way for seccomp exceptions, you need to use `seccomp.drop`
